org.eclipse.scout.rt.server.commons/src/main/java/org/eclipse/scout/rt/server/commons/healthcheck/AbstractHealthChecker.java

```diff
@@ -64,14 +64,14 @@ protected long getConfiguredTimeToLiveMillis() {
   }
 
   /**
-   * @return The health check timeout duration in milliseconds. If greater than zero, {@link #execCheckHealth()} will
-   *         time out after given duration.
+   * @return The health check timeout duration in milliseconds. If greater than zero,
+   *         {@link #execCheckHealth(HealthCheckCategoryId)} will time out after given duration.
    */
   protected long getConfiguredTimeoutMillis() {
     return 0;
   }
 
-  protected abstract boolean execCheckHealth() throws Exception;
+  protected abstract boolean execCheckHealth(HealthCheckCategoryId category) throws Exception;
```

Comment:
Add to releasenotes/migration guide -> API change for health checks

Reply:
Fixed in https://github.com/bsi-software/org.eclipse.scout.docs/pull/127